tests/NovelRT.Tests/CMakeLists.txt

```diff
@@ -56,18 +56,19 @@ set(SOURCES
 add_executable(Engine_Tests ${SOURCES})
 
 if(NOVELRT_BUILD_INTEROP)
-    copy_build_products(Engine_Tests
-      DEPENDENCY Engine
-      TARGET_LOCATION $<TARGET_FILE_DIR:Engine_Tests>
+  copy_build_products(Engine_Tests
+    DEPENDENCY Engine
+    TARGET_LOCATION $<TARGET_FILE_DIR:Engine_Tests>
 
-      DEPENDENCY Interop
-      TARGET_LOCATION $<TARGET_FILE_DIR:Engine_Tests>
-    )
+    DEPENDENCY Interop
+    TARGET_LOCATION $<TARGET_FILE_DIR:Engine_Tests>
+  )
 else()
-    copy_build_products(Engine_Tests
-            DEPENDENCY Engine
-            TARGET_LOCATION $<TARGET_FILE_DIR:Engine_Tests>
-    )
+  include(${CMAKE_SOURCE_DIR}/cmake/CopyBuildProducts.cmake)
+  copy_build_products(Engine_Tests
+    DEPENDENCY Engine
+    TARGET_LOCATION $<TARGET_FILE_DIR:Engine_Tests>
+  )
```

Comment:
I think this is also just spacing (and the include that was probably missing?) But just wanted to check.

Reply:
Yeeee